update_prevalence.py

```diff
@@ -235,6 +288,66 @@ def cases_week_before(self) -> int:
     def app_key(self) -> str:
         ...
 
+    def set_total_vaccines(self, partial_vaccinations: int, complete_vaccinations: int):
+        self.vaccines_total.partial_vaccinations = partial_vaccinations
+        self.vaccines_total.completed_vaccinations = complete_vaccinations
+    
+    def set_vaccines_of_type(self, vaccine_type: str, partial: int, complete: int):
+        if self.vaccines_by_type is None:
+            self.vaccines_by_type = {}
+        
+        self.vaccines_by_type[vaccine_type] = Vaccination()
+        self.vaccines_by_type[vaccine_type].partial_vaccinations = partial
+        self.vaccines_by_type[vaccine_type].completed_vaccinations = complete
+
+    def completed_vaccination_total(self):
+        if (self.vaccines_by_type is not None):
+            return reduce(lambda x, key: x + self.vaccines_by_type[key].completed_vaccinations, self.vaccines_by_type, 0)
+        return self.vaccines_total.completed_vaccinations
+
+    def partial_vaccination_total(self):
+        if (self.vaccines_by_type is not None):
+            return reduce(lambda x, key: x + self.vaccines_by_type[key].partial_vaccinations, self.vaccines_by_type, 0)
+        return self.vaccines_total.partial_vaccinations
+
+    # Compute the estimated risk ratio for an unvaccinated person vs an average person.
+    # Average prevalence = sum(prevalence in group * proportion of population in group)
+    #                    = sum(vaccine_mult * unvaccinated_prev * proportion of population)
+    # Unvaccinated risk / Average risk = sum(vaccine_mult * proportion of pop)
+    #                                  = population / sum(vaccine_mult * # of vaccinations)
```

Comment:
I think this should be `sum(vaccine_mult * # of vaccinations) + num_unvaccinated`

Reply:
It is; I consider "unvaccinated" to be a vaccination state. Modified to be clearer.